ps_mbo.php

```diff
@@ -310,6 +310,13 @@ public function uninstallTab(array $tabData)
      */
     public function hookActionAdminControllerSetMedia()
     {
+        if (strpos(_PS_VERSION_, '8') === 0) {
```

Comment:
We could use 
if (version_compare(_PS_VERSION_, '8.0.0') >= 0) {
to be larger but it's fine like that ;)